src/lib/crypto/kimchi_backend/common/plonk_dlog_proof.ml

```diff
@@ -229,28 +229,31 @@ module Make (Inputs : Inputs_intf) = struct
     ; challenge_polynomial_commitment = g t.sg
     }
 
+  let lookup_eval_of_backend
+      ({ sorted; aggreg; table; runtime } : 'f Kimchi_types.lookup_evaluations)
+      : _ Pickles_types.Plonk_types.Evals.Lookup.t =
+    { sorted; aggreg; table; runtime }
```

Comment:
isn't it enough to re-export these types?

```ocaml
module Pickles_types.Plonk_types.Evals.Lookup = struct
   type t = 'f Kimchi_types.lookup_evaluations = { sorted: _; aggreg: _; table: _; runtime: _ }
```

Reply:
Yea, I didn't want to go through the routing though. TODO later